pages/osx/mdutil.md

```diff
@@ -0,0 +1,19 @@
+# mdutil
+
+> Manage the metadata stores used by Spotlight for indexing.
+
+- Show the indexing status of the startup volume:
+
+`mdutil -s {{/}}`
+
+- Turn off the Spotlight indexing for the volume mounted on /Volumes/Foo:
```

Comment:
Possibly merge this with the one below?
```
mdutil -i {{on|off}} {{path/to/volume}}
```

Also, is `/Volumes` an example storage directory? :slightly_smiling_face:

Reply:
Merging the two examples is a very good suggestion, thanks! 
As for the second question, do you mean that /Volumes/Foo was not a good example?

Reply:
We usually use `path/to/volume` unless a concrete directory makes an example clearer. If /Volumes/Foo is just a normal mounted path, I think path/to/volume is a better choice.